modules/install/pages/upgrade.adoc

```diff
@@ -1,73 +1,94 @@
+then make sure your cluster is *not* running in `mixed mode`.
+(i.e. some nodes are using IPv4 addressing while some are using IPv6).
 ====
 
 [#understanding-upgrade]
 == Understanding Upgrade
 
-To _upgrade_ a Couchbase-Server cluster means to revise upwards the version of the server that is running on every node.
-For example, a cluster each of whose nodes has been running Couchbase Server Enterprise Edition Version 6.6 is modified; such that each of its nodes subsequently runs Couchbase Server Enterprise Edition Version 7.2.
+To _upgrade_ a Couchbase-Server cluster means to upgrade the version of the server that is running on every node.
+For example, modifying a cluster where all of its nodes are running Couchbase Server Enterprise Edition Version 6.6,
+so that each of its nodes subsequently runs Couchbase Server Enterprise Edition Version 7.2.
 
-An _upgrade procedure_, like an install procedure, involves both preparation routines and specific upgrade commands that are performed on each individual node.
-To be upgraded, a cluster must have each of its nodes individually upgraded in turn; and the upgrade procedure for the cluster must therefore be selected with regards to whether the cluster is required to continue serving data, or to cease serving data, during the course of the cluster-upgrade.
+An _upgrade procedure_, like an _install_ procedure, involves both preparation routines and specific upgrade commands that are performed on each node.
+To be upgraded, a cluster must have each of its nodes individually upgraded in turn; and the upgrade procedure for the cluster must therefore be selected in regard to whether the cluster is required to continue serving data, or to cease serving data, during the cluster-upgrade.
 A review of the factors that determine the appropriateness of an upgrade-procedure is provided in xref:install:upgrade-procedure-selection.adoc[Upgrade Procedure-Selection].
 
 [#supported-upgrade-paths]
 == Upgrade Paths
 
 An upgrade _path_ declares that the upgrade of one version of Couchbase Server to another is _supported_.
 The tables in the following subsections list upgrade paths for Enterprise Edition and for Community Edition, respectively.
-Each instance of the sign -> declares support for the upgrade of the server-version on the left of the sign to the server-version on the right.
-Note that upgrade between non-adjacent version-numbers is frequently _not_ supported: for example, to upgrade from 5.5.0+ to 7.2, upgrade must be performed _twice_; first from 5.5.0+ to 6.6, and secondly from 6.6 to 7.2.
+Each instance of the{nbsp}`->`{nbsp}sign declares support for the upgrade of the server-version on the left of the sign to the server-version on the right.
+
+.Upgrading between non-adjacent version numbers is usually _not_ supported.
+[NOTE]
+====
+For example, to upgrade from *5.5.0+* to *7.2*, then _two_ upgrades must be performed:
+first, from** 5.5.0+** to** 6.6**,
+and then secondly, from *6.6* to *7.2*.
```

Comment:
This is unfortunately a bad example, because upgrade from 6.6 directly to 7.2 is _not_ supported (this is assuming "7.2" means "latest 7.2.x" which is now 7.2.4).